### PR TITLE
Version 1.3.0

### PR DESCRIPTION
**Pull request by fridex**

### CODE REVIEW

**Comment by fridex** [comment]
/assign @frenzymadness 

PTAL

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,23 @@
 # Changelog for Thoth's micropipenv
 
+## [1.3.0] - 2022-April-08 - Fridolin Pokorny <fridolin@redhat.com>
+
+### Added
+
+* Improve poetry → requirements and handling of transitive deps
+  Contribution thanks to @frenzymadness
+
+* Improve handling of poetry → requirements
```
Comment (frenzymadness) [inline CHANGELOG.md:10]: Isn't this point already included in the previous one?

**Review by frenzymadness — COMMENTED** [review]
I'd also add to the changelog that this release is tested with pip 22.0.4 but it's up to you.

File: CHANGELOG.md (hunk 1 above)
Comment (fridex) [reply CHANGELOG.md]: Thanks for the review, adjusted 👍🏻

**Review by frenzymadness — APPROVED** [review]

**Pull request merged**